<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wVisitorsTree</class>
 <widget class="QWidget" name="wVisitorsTree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="visitorsTree">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="indentation">
      <number>25</number>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>160</number>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>160</number>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string comment="LBL_VISITORS_TREE_IDLE">Idle</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>227</red>
         <green>227</green>
         <blue>227</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
      <property name="icon">
       <iconset resource="lh3.qrc">
        <normaloff>:/lh3/Resources/collapsed.png</normaloff>
        <normalon>:/lh3/Resources/expanded.png</normalon>:/lh3/Resources/collapsed.png</iconset>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>(0)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>227</red>
         <green>227</green>
         <blue>227</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="LBL_VISITORS_TREE_WAITING_CHAT">Waiting for chat</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>217</green>
         <blue>195</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
      <property name="icon">
       <iconset resource="lh3.qrc">
        <normaloff>:/lh3/Resources/collapsed.png</normaloff>
        <normalon>:/lh3/Resources/expanded.png</normalon>:/lh3/Resources/collapsed.png</iconset>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>(0)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>217</green>
         <blue>195</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="LBL_VISITORS_TREE_CHATTING">Chatting</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>230</green>
         <blue>157</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
      <property name="icon">
       <iconset resource="lh3.qrc">
        <normaloff>:/lh3/Resources/collapsed.png</normaloff>
        <normalon>:/lh3/Resources/expanded.png</normalon>:/lh3/Resources/collapsed.png</iconset>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>(0)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>230</green>
         <blue>157</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="LBL_VISITORS_TREE_INVITATION_SENT">Invitation sent</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>211</green>
         <blue>134</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
      <property name="icon">
       <iconset resource="lh3.qrc">
        <normaloff>:/lh3/Resources/collapsed.png</normaloff>
        <normalon>:/lh3/Resources/expanded.png</normalon>:/lh3/Resources/collapsed.png</iconset>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>(0)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>211</green>
         <blue>134</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="LBL_VISITORS_TREE_INVITATION_DECLINET">Invitation declined</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>231</green>
         <blue>174</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
      <property name="icon">
       <iconset resource="lh3.qrc">
        <normaloff>:/lh3/Resources/collapsed.png</normaloff>
        <normalon>:/lh3/Resources/expanded.png</normalon>:/lh3/Resources/collapsed.png</iconset>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="text">
       <string>(0)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>244</red>
         <green>231</green>
         <blue>174</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>72</red>
         <green>72</green>
         <blue>72</blue>
        </color>
       </brush>
      </property>
     </item>
    </widget>
   </item>
  </layout>
  <action name="actionAssign_name">
   <property name="text">
    <string>Assign name</string>
   </property>
  </action>
  <action name="actionOpen_Visitor">
   <property name="text">
    <string>Open Visitor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="lh3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>visitorsTree</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>wVisitorsTree</receiver>
   <slot>itemDoubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>visitorsTree</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>wVisitorsTree</receiver>
   <slot>showPopup(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAssign_name</sender>
   <signal>activated()</signal>
   <receiver>wVisitorsTree</receiver>
   <slot>renameVisitorAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Visitor</sender>
   <signal>activated()</signal>
   <receiver>wVisitorsTree</receiver>
   <slot>openVisitorAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>itemDoubleClicked(QTreeWidgetItem*,int)</slot>
  <slot>itemClicked(QTreeWidgetItem*,int)</slot>
  <slot>itemSelectionChanged()</slot>
  <slot>showPopup(QPoint)</slot>
  <slot>openVisitorAction()</slot>
  <slot>renameVisitorAction()</slot>
 </slots>
</ui>
